src/pomodoro.sh

```diff
@@ -49,6 +49,16 @@ function pomodoro_main()
   if [[ -n "${options_values['TIMER']}" ]]; then
     timer_thread "$flag" &
   fi
+
+  if [[ -n "${options_values['WATCH']}" ]]; then
+    clear
+    while true; do
+      tput cup 0 0
+      show_active_pomodoro_timebox "$flag"
+      sleep 1
```

Comment:
Since you are locking the terminal, it would be nice to handle the Ctrl + C scenario in which the user cancels this operation.

Reply:
Thanks for noticing that! I realized that when Ctrl+C is executed, the timer thread is killed too. That is not the desired behavior right?